integration-tests/nats/src/test/java/org/apache/camel/quarkus/component/nats/it/NatsTestResource.java

```diff
@@ -63,6 +65,20 @@
         Integer noAuthPort = noAuthContainer.getMappedPort(NATS_SERVER_PORT);
         String noAuthBrokerUrl = String.format("%s:%s", noAuthIp, noAuthPort);
 
+        // Start the container needed for the TLS authentication test
+        tlsAuthContainer = new GenericContainer(NATS_IMAGE).withExposedPorts(NATS_SERVER_PORT)
+                .withClasspathResourceMapping("certs", "/certs", BindMode.READ_ONLY)
```

Comment:
It uses `ClassLoader.getResource(...)` under the hood, so I think the test could be executed on platform too.